### ISSUE TITLE
Subscribe to keyboard visibility is not working with floating keyboard

### ISSUE THREAD

**Issue by AhmedMansour92** [MisterJimson/flutter_keyboard_visibility#114]
Subscribe to keyboard visibility is not working with floating keyboard , not detecting changes

**Comment by AhmedMansour92** [comment]
yes in android, i have no screenshots as i've removed it from my project , it was working fine for all scenarios unless the floating keyboard.

**Comment by aj-Thapa** [comment]
I was searching for the same: detection of floating keyboard. Also I can't find any information regarding the detection of keyboard at different edges of the screen. Does it detect keyboard only at the bottom of the screen? Somewhere, I have also seen keyboard on the right hand side of the screen while the app page displayed on the left side. Does it detect such layout as well? I request you to include which keyboard display "layout" are supported (bottom, sides, floating, etc.) in your documentation - the Readme section of `pub.dev` clearly so that we are well informed. Until then I would be thankful if someone could answer it here too for me.
